.github/scripts/publish.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh
+set -e
+
+# List of packages to prevent from publishing
+EXCLUDED_PACKAGES=(
+  "storybook"
+)
+
+# List of packages in the workspace
+WORKSPACE_PACKAGES=$(jq -r '.workspaces[]' package.json)
+WORKSPACE_PACKAGES_ARRAY=($WORKSPACE_PACKAGES)
+
+# We substract the excluded packages from the workspace packages
+PACKAGES=()
+for package in "${WORKSPACE_PACKAGES_ARRAY[@]}"; do
+    if [[ ! " ${EXCLUDED_PACKAGES[@]} " =~ " ${package} " ]]; then
+        PACKAGES+=("$package")
+    fi
+done
+
+# Print the filtered list of packages
+echo "::group::Packages to publish"
+echo "${PACKAGES[@]}"
+echo "::endgroup::"
+
+# Get the version from the tag
+VERSION=${GITHUB_REF/refs\/tags\//}
+
+# Helper function to update the version and publish the package
+# Note : we are in the CI triggered by a release, we do not need to create a tag nor commit
+version_and_publish() {
+  local dir=$1
+  local version=$2
+
+  # Go to the package directory
+  pushd $dir
```

Comment:
pushd/popd aren't posix and aren't needed here. Use `cd $dir` and `cd -` instead